### ISSUE TITLE
Calling EpwFile::designConditions() results in twice as much EpwFile::data as there should be

### ISSUE THREAD

**Issue by joseph-robertson** [NREL/OpenStudio#3215]
Brought this up with Jason. He says: 

It's calling parse here, and I think that's the problem
(with true as the second arg)
We're going to need to change that and probably code in the data method to avoid double reading the design conditions

**Comment by jasondegraw** [comment]
@joseph-robertson @shorowit Since we haven't been able to reproduce the error completely in C++, I'm going to fix the problem that I can cause, and that's double reading of design conditions. PR should hit within the next couple hours.

**Comment by shorowit** [comment]
I was able to reproduce the original issue using the attached EPW. Not sure if it's because the EPW has no design conditions or because it has a leap day in it?

```
require 'openstudio'
m = OpenStudio::Model::Model.new
epw_file = OpenStudio::EpwFile.new("C:/GitHub/OpenStudio-BEopt/test.epw")
epw_file.data.size # => 8784
epw_file.designConditions.size # => 0
epw_file.data.size # => 17568 (BAD)
```

[test.epw.txt](https://github.com/NREL/OpenStudio/files/2430661/test.epw.txt)

**Comment by jasondegraw** [comment]
@shorowit It's the lack of design conditions, weird things can happen with leap days but this shouldn't be one. I'm pretty sure that no design conditions would have resulted in repeated calls to the `parse` member function with the data read argument set to `true`. I'm pretty sure that's fixed in my PR, but to be sure I'll add an additional test later today or tomorrow.

**Comment by jasondegraw** [comment]
@shorowit I've added a new unit test over in #3288 using your EPW, and the data is no longer doubled up when there are no design conditions

**Comment by shorowit** [comment]
Great, thanks @jasondegraw!

**Comment by jasondegraw** [comment]
This was addressed in #3288
